config/initializers/pipeline.rb

```diff
@@ -0,0 +1,19 @@
+module IdSeqPipeline
+  AWS_ACCESS_KEY_ID = ENV['AWS_ACCESS_KEY_ID']
```

Comment:
We are going to be using instance profiles to configure aws in dev/stage/prod. Locally we should use aws credentials profiles, rather than environment variable.s

Reply:
we can worry about this later tho

Reply:
how do we do that inside a docker container?

Reply:
this is fine for now